en/docs/reference/config-catalog.md

```diff
@@ -3913,7 +3913,12 @@ client_registration_password = "$ref{super_admin.password}"</code></pre>
                 <label class="tab-selector" for="_tab_27"><i class="icon fa fa-code"></i></label>
                 <div class="superfences-content">
                     <div class="mb-config-example">
-                        <pre><code class="toml">#SAMPLE</code></pre>
+                        <pre><code class="toml">[apim.sdk]
+group_id = "org.wso2"
+artifact_id = "org.wso2.client"
+model_package = "org.wso2.client.model"
+api_package = "org.wso2.client.api"
+supported_languages = ["android", "java", "scala", "csharp", "dart", "flash", "groovy", "javascript"]

```

Comment:
Shall we format/align these properly?

Reply:
We are generating this through https://github.com/wso2/docs-apim/tree/3.2.0/en/tools/config-catalog-generator. This is how they are generated.